README.md

```diff
@@ -65,9 +65,56 @@ recommended Java logging API within Google.
 
 ### 1. Add the dependencies on Flogger
 
-All code that uses flogger should depend on
-`com.google.flogger:flogger:<version>` and
-`com.google.flogger:flogger-system-backend:<version>`.
+All code that uses flogger should depend on:
+
+* `com.google.flogger:flogger:<version>`
+* `com.google.flogger:flogger-system-backend:<version>`
+
+**Maven**
+
+```xml
+    <dependency>
+      <groupId>com.google.flogger</groupId>
+      <artifactId>flogger</artifactId>
+      <version>${flogger.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.flogger</groupId>
+      <artifactId>flogger-log4j-backend</artifactId>
+      <version>${flogger.version}</version>
+      <scope>runtime</scope>
+    </dependency>
+```
+
+Optionally
```

Comment:
Can you please describe in what circumstances one would want to add the explicit dependency on log4j/exclusions? If I'm reading this directly, I don't know why/when I would do this instead of the first example above.

If the explanation is long enough, perhaps a footnote would help?

Reply:
@nglorioso I don't know / remember. Flogger wasted more time than it saved to the point where we ended up debugging JUL in tears.

It manages to take something that typically causes problems (e.g. slf4j + X) and compounds that complexity into something more difficult. I'm not sure it's worth the trouble. 

I may be too stupid / ignorant to actually figure it out. Currently  have to do static blocks like 

```
  static {
    System.setProperty("flogger.backend_factory", "com.google.common.flogger.backend.slf4j.Slf4jBackendFactory#getInstance");
  }

``` to get it working but for some reason this PR is still open...

Here's how the dependency management section looks like

```
      <dependency>
        <groupId>com.google.flogger</groupId>
        <artifactId>flogger</artifactId>
        <version>${google.flogger.version}</version>
      </dependency>
      <dependency>
        <groupId>com.google.flogger</groupId>
        <artifactId>flogger-system-backend</artifactId>
        <version>${google.flogger.version}</version>
        <scope>runtime</scope>
      </dependency>
      <dependency>
        <groupId>com.google.flogger</groupId>
        <artifactId>flogger-slf4j-backend</artifactId>
        <version>${google.flogger.version}</version>
      </dependency>
      <dependency>
        <groupId>org.slf4j</groupId>
        <artifactId>slf4j-api</artifactId>
        <version>${slf4j.version}</version>
      </dependency>
      <dependency>
        <groupId>ch.qos.logback</groupId>
        <artifactId>logback-classic</artifactId>
        <version>${logback.version}</version>
      </dependency>
      <dependency>
        <groupId>ch.qos.logback</groupId>
        <artifactId>logback-core</artifactId>
        <version>${logback.version}</version>
      </dependency>
```